docs/source/install.md

```diff
@@ -71,13 +75,18 @@ $ conda install -c conda-forge pyspark
 ```console
 $ conda install -c conda-forge alteryx-open-src-update-checker
 ```
+```{tab} Featuretools_SQL
+```console
+$ conda install -c conda-forge featuretools_sql
```

Comment:
Should I keep this here, or wait until we add `featuretools_sql` to conda-forge?

Reply:
Let's wait to add this